packages/wallet/api/src/lib-wallet.js

```diff
@@ -957,7 +962,7 @@ export function makeWallet({
           origin === null ||
           (offer.requestContext && offer.requestContext.dappOrigin === origin),
       )
-      .sort(([id1], [id2]) => cmp(id1, id2))
+      .sort(([id1], [id2]) => id1.localeCompare(id2))
```

Comment:
Is `localeCompare` sufficiently deterministic? Doe it depend on an ambient locale setting? Looks like it's OK...

## Details

It took me a while to confirm, so I might as well save my notes:

> Much of the `Intl` package, and some locale-specific portion of other objects (e.g. Number.prototype.toLocaleString) has results which depend upon which locale is configured, and that will vary from one process to another. The way we handle this is still in development. Either these functions will be disabled, or they will act as if they run on a host with a single fixed locale defined by the SES specification.
-- https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/vat-environment.md

I also did a quick experiment to confirm that XS supports it by adding this to `packages/xsnap/test/test-xs-js.js`.

```js
test('String.prototype.localeCompare', async t => {
  const opts = options(io);
  const vat = xsnap(opts);
  await vat.evaluate(`
  const encoder = new TextEncoder();
  const send = it => issueCommand(encoder.encode(JSON.stringify(it)).buffer);
  send('abc'.localeCompare('def') < 0)
  `);
  t.deepEqual(opts.messages, ['true']);
  await vat.terminate();
});
```

Reply:
Thanks. Added in https://github.com/Agoric/agoric-sdk/pull/5873